ui/tests/acceptance/access/namespaces/index-test.js

```diff
@@ -0,0 +1,40 @@
+import { currentRouteName } from '@ember/test-helpers';
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import page from 'vault/tests/pages/access/namespaces/index';
+import authPage from 'vault/tests/pages/auth';
+import logout from 'vault/tests/pages/logout';
+
+module('Acceptance | /access/namespaces', function(hooks) {
```

Comment:
Need to add `Enterprise` to this module name, since namespaces are enterprise only

Reply:
You saved my day...Thanks!! 😅